core/src/main/java/com/orientechnologies/orient/core/record/impl/ODocument.java

```diff
@@ -128,17 +128,17 @@
 
   protected Map<String, ODocumentEntry> fields;
 
-  protected           boolean                             trackingChanges        = true;
```

Comment:
This could probably be a separate refactoring...

Reply:
this was part of a separate refactoring, i pushed it separately now is not part of this PR